nibabel/processing.py

```diff
@@ -0,0 +1,296 @@
+# emacs: -*- mode: python-mode; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the NiBabel package for the
+#   copyright and license terms.
+#
+### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+""" Image processing functions for:
+
+* smoothing
+* resampling
+* converting sd to and from FWHM
+
+Smoothing and resampling routines need scipy
```

Comment:
I would like to leave this out of the docs for now, to avoid people hitting the problem with unspecified axes.